algorithms/number_theory/eratosthenes_sieve.py

```diff
@@ -0,0 +1,59 @@
+"""Module with the implementation of eratosthenes sieve."""
+
+
+from typing import List
+
+
+class EratosthenesSieve:
+    """Class implementation of the eratosthenes sieve algorithm."""
+
+    def __init__(self, n: int):
+        self._n = n
+        self._no_cousins = [False for _ in range(self._n + 1)]
+
+    def eratosthenes_sieve(self) -> List[bool]:
+        """
+        Eratosthenes sieve algorithm.
+        """
+        self._no_cousins[0] = self._no_cousins[1] = True
+
+        for i in range(2, self._n + 1):
+            if not self._no_cousins[i]:
+                for j in range(i, self._n + 1):
+                    if i * j >= self._n + 1:
+                        break
```

Comment:
I think this can be rewritten as:

```
for j in range(i + i, self._n + 1, i):
    self._primes[j] = True
```

When we make the name changes to the `_no_cousins` list.

This way we get rid of the `break` statement and the code is more clear. Agree?

Reply:
agree